R/layouts.R

```diff
@@ -251,3 +251,27 @@ split_layout <- function(..., cell_widths = NULL, cell_args = "", style = NULL){
 splitLayout <- function(..., cellWidths = NULL, cellArgs = "", style = NULL) {
   split_layout(..., cell_widths = cellWidths, cell_args = cellArgs, style = style)
 }
+
+#' @export
+#' @rdname flow_layout
+flow_layout <- function(..., cellArgs = list(), itemWidth = "208px", columnGap = "12px", rowGap = "0px") {
```

Comment:
please change argument names to `snake_case` we use camelCase only for wrappers to be consistent with shiny syntax

Reply:
Done.